docs/reference/configuration/UDS operator/package.md

```diff
@@ -10,6 +10,10 @@ sidebar:
 
 - **Enabling Istio Sidecar Injection:**
   - The operator facilitates the activation of Istio sidecar injection within namespaces where the CR is deployed.
+- **Support for Istio Ambient Mode:**
+  - Packages can now opt into Istio's Ambient mode by setting `spec.network.serviceMesh.mode: ambient`. This provides service mesh capabilities without sidecars, reducing resource overhead.
+  - If Ambient mode is not deployed in the cluster, the operator will automatically fall back to sidecar mode.
+  - Note that `Packages` with Authservice clients are not currently supported in ambient mode.
```

Comment:
This is probably a crucial information and I also see we fail if a user tries to deploys an unsupported configuration. Can we somehow make it easier to spot in the documentation? Maybe we should put it in a "Limitations" paragraph or something similar.

Reply:
Added this to the Authservice limitations section in the SSO Docs - let me know if you think this or other lines should be present in a different location as well.